tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/core/task_generator.h

```diff
@@ -23,8 +23,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#ifndef TESSERACT_PROCESS_MANAGERS_PROCESS_GENERATOR_H
-#define TESSERACT_PROCESS_MANAGERS_PROCESS_GENERATOR_H
+#ifndef TESSERACT_PROCESS_MANAGERS_task_generator_H
```

Comment:
I assume this happened during copy replace. Should be able to do a replace _task_generator_H with _TASK_GENERATOR_H.

Reply:
I will be pushing a commit to address the remaining items.